src/element-style-observer.js

```diff
@@ -109,7 +109,29 @@ export default class ElementStyleObserver {
 	 * @param {TransitionEvent} event
 	 */
 	async handleEvent (event) {
-		if (!this.properties.has(event.propertyName)) {
+		if (event.type === "animationstart") {
+			let animation = document.getAnimations().find(a => a.effect.target === this.target);
+			let properties = this.getObservedAnimationProperties(animation);
+			if (properties.length === 0) {
+				return;
+			}
+			else {
+				let interval = setInterval(() => this.#handleChanges(properties), 16); // ~60fps
```

Comment:
We should probably support a separate `animationThrottle` setting

Reply:
Agreed. Updated the code.